packages/contracts/src/token/CBL.sol

```diff
@@ -0,0 +1,30 @@
+//SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ERC20Permit } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+import { ERC20Burnable } from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import { Pausable } from "@openzeppelin/contracts/utils/Pausable.sol";
+
+contract CBL is ERC20, Ownable, ERC20Permit, ERC20Burnable, Pausable {
```

Comment:
are we planing to have Ownable or AccessControl?

Reply:
should be AccessControl.  i will update.